examples/backend/package.yaml

```diff
@@ -9,15 +9,15 @@ copyright:           "Copyright 2022 Chaos Software OOD"
 extra-source-files:
 - README.md
 
-description:         Please see the README on GitHub at <https://github.com/githubuser/servant-elm-street-main#readme>
+description:         Please see the README on GitHub at <https://github.com/githubuser/backend#readme>
 
 dependencies:
 - base >= 4.7 && < 5
-- text
+- elm-street
 
 ghc-options:
-- -Wall
 - -Werror
+- -Wall
```

Comment:
Try to avoid unnecessary shuffling like this in the future.

Reply:
okay